app/models/service/aggregation.rb

```diff
@@ -315,7 +315,7 @@ def aggregate_all_vm_disk_space_used
   def aggregate_all_vm_memory_on_disk
     if has_attribute?("aggregate_all_vm_memory_on_disk")
       # Avoids (poorly) "ERROR:  integer out of range" in postgres
-      self["aggregate_all_vm_memory_on_disk"] * 1.megabyte
+      self["aggregate_all_vm_memory_on_disk"].try(:*, 1.megabyte)
```

Comment:
@bdunne The original PR this comes from is https://github.com/ManageIQ/manageiq/pull/11502 and might provide context as to what it is "doing" (or at least what all the extra junk around it is).

Specifically, the `aggregate_all_vm_memory_on_disk` method is a `virtual_attribute` that basically finds all of the ram being used actively by running VMs (from what I understand), but is defined here:  https://github.com/NickLaMuro/manageiq/blob/99ff33a/app/models/service/aggregation.rb#L22-L23

The helper method that used to define the arel for that, [`aggregate_service_hardware_arel`](https://github.com/NickLaMuro/manageiq/blob/99ff33a/app/models/service/aggregation.rb#L38-L48), should have probably taken some cues from [`aggregate_all_vms_disk_space_allocated_arel`](https://github.com/NickLaMuro/manageiq/blob/99ff33a/app/models/service/aggregation.rb#L61-L72) and [`aggregate_all_vms_disk_space_used_arel`](anageiq/blob/99ff33a/app/models/service/aggregation.rb#L74-L85) and wrapped the `SUM` in a `COALESCE` to prevent the nil from being possible.  But again, probably something for a different PR as that is hit more than one of these scopes to make that change.

Regarding the tests, that mostly falls on me since I didn't write them in the original PR.  That said, I think the types of tests we would want in the long run would be very involved and require a lot of setup since these are aggregating results, which is something I don't mind doing as a follow up to this.  Would something like this make sense for now?

⚠️ WARNING:  PSEUDO CODE DETECTED ⚠️ 

```ruby
it "doesn't blow up when nil" do
  expect(described_class).to receive(:has_attribute?).with("aggregate_all_vm_memory_on_disk")
  expect{described_class.new.aggregate_all_vm_memory_on_disk}.to_not raise_error(NoMethodError)
end
```